style.css

```diff
@@ -159,3 +159,323 @@
   right: 0;
   top: 87.09%;
 }
+
+.work {
+  background-color: rgb(229 229 229);
+  padding: 120px 64px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+}
+
+.work-title1 {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-bottom: 30px;
+}
+
+.work-head {
+  width: 327px;
+  height: 83px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+  display: flex;
+  align-items: center;
+  padding-bottom: 10px;
+  border-bottom: 1px solid black;
+  margin-bottom: 50px;
+}
+
+.work-head2 {
+  width: 327px;
+  height: 36px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
```

Comment:
[ ] kindly use the same font defined for the template🤓.
![fonty](https://user-images.githubusercontent.com/62156607/198576766-3116e12a-9c09-4ab7-9015-7c1c7b55fd07.PNG)